### PR TITLE
update contribution readme fixes#4

### PR DESCRIPTION
**Pull request by okothkongo**
this guide contributors  on how to do so with ease and meet our requirements

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,15 +1,52 @@
...
+
+`3-update-contribution-readme`
+
+Run 
+
+```mix check_lints``` and    ```mix test```  and solve any arising issues before pushing
+
+Upon making changes push to your remote 
+
+```$git push origin branchname```
```
Comment (FedericoEsparza) [inline README.md:45]: Let's encourage people to push to `upstream`

```$ git push upstream branchname```

Comment (okothkongo) [reply README.md:45]: then there is no need for forking?. my idea was they fork, push to their repo and create a pull request to ours. They will be having two remotes upstream(ours) and origin(their's)

File: README.md [hunk 2]
```diff
@@ -1,15 +1,52 @@
...
+
+```$git clone https://github.com/your_username/ReTither.git```  
+
+or   
+```$git clone git@github.com:your_username/ReTither.git```
+
 
 To start your Phoenix server:
 
   * Install dependencies with `mix deps.get`
```
Comment (FedericoEsparza) [inline README.md:14]: We might add a line above this, like so: 
```
* Change directory into the cloned repo: `cd ReTither`
```

File: README.md (hunk 1 above)
Comment (FedericoEsparza) [reply README.md:45]: Let's not waste time on documentation, yet. Good job getting this done.

People might fork it, to get started right away, while they wait for us to give them collaborator access. Therefore the fork is ok.

**Review by 75pollet — APPROVED** [review]

**Review by aonomike — APPROVED** [review]
I think we can merge this one. More updates will be made as we progress. Thanks @okothkongo1

**Review by FedericoEsparza — APPROVED** [review]
@okothkongo1 Thanks for getting this one done, Okoth.

**Pull request merged**